Adds Director of ML Insights Part 2 (#288)
huggingface/blog: ml-director-insights-2.md

---
title: "Director of Machine Learning Insights [Part 2: SaaS Edition]"
thumbnail: /blog/assets/67_ml_director_insights/thumbnail.png
---

<html>
<head>
<style>
.grandmahugs {
  margin: 25px;
}
</style>
<h1>Director of Machine Learning Insights [Part 2: SaaS Edition]</h1>

<div class="blog-metadata">
    <small>Published May 13, 2022.</small>
</div>

<div class="author-card">
    <a href="/britneymuller">
        <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1645809068511-5ef0ce775e979253a010ef4c.jpeg?w=200&h=200&f=face" title="Gravatar">
        <div class="bfc">
            <code>britneymuller</code>
            <span class="fullname">Britney Muller</span>
        </div>
    </a>
</div>
</head>
<body>

_If you or your team are interested in building ML solutions faster visit [hf.co/support](https://huggingface.co/support?utm_source=article&utm_medium=blog&utm_campaign=ml_director_insights_2) today!_

> 👋 Welcome to Part 2 of our Director of Machine Learning Insights [Series]. Checkout [Part 1 here.](https://huggingface.co/blog/ml-director-insights)

Reviewer: ```suggestion
👋 Welcome to Part 2 of our Director of Machine Learning Insights [Series]. Check out [Part 1 here.](https://huggingface.co/blog/ml-director-insights)
```

> Directors of Machine Learning have a unique seat at the AI table spanning the perspective of various roles and responsibilities. Their rich knowledge of ML frameworks, engineering, architecture, real-world applications and problem-solving provides deep insights into the current state of ML. For example, one Director will note how using new transformers speech technology decreased their team’s error rate by 30% and how simple thinking can help save _a lot_ of computational power.

Reviewer 2: ```suggestion
Directors of Machine Learning have a unique seat at the AI table spanning the perspective of various roles and responsibilities. Their rich knowledge of ML frameworks, engineering, architecture, real-world applications and problem-solving provides deep insights into the current state of ML. For example, one director will note how using new transformers speech technology decreased their team’s error rate by 30% and how simple thinking can help save _a lot_ of computational power.
```

> In this second SaaS focused installment, you’ll hear from a deep learning for healthcare Author who founded a non-profit for mentoring ML talent, a chess fanatic Cybersecurity expert, an Entrepreneur whose business was inspired by Barbie’s need to monitor brand reputation after a lead recall, and a seasoned patent and academic paper author who enjoys watching his 4 kids make the same mistakes as his ML models.  

Reviewer 2: ```suggestion
In this second SaaS focused installment, you’ll hear from a deep learning for healthcare textbook author who also founded a non-profit for mentoring ML talent, a chess fanatic cybersecurity expert, an entrepreneur whose business was inspired by Barbie’s need to monitor brand reputation after a lead recall, and a seasoned patent and academic paper author who enjoys watching his 4 kids make the same mistakes as his ML models.  
```

Reviewer 2: I would lowercase these words here (seems a bit odd to me that these words are capitalized).

Reviewer 2: also wondering to just say _"deep learning for healthcare textbook author"_ and remove _"who also founded a non-profit for mentoring ML talent"_ to keep it simple / easier to read

Directors of Machine Learning have a unique seat at the AI table spanning the perspective of various roles and responsibilities. Their rich knowledge of ML frameworks, engineering, architecture, real-world applications and problem-solving provides deep insights into the current state of ML. For example, one director will note how using new transformers speech technology decreased their team’s error rate by 30% and how simple thinking can help save _a lot_ of computational power.

> Ever wonder what Directors at Salesforce or ZoomInfo currently think about the state of Machine Learning? What their biggest challenges are? And what they're most excited about? Well, you're about to find out!

Reviewer 2: ```suggestion
Ever wonder what directors at Salesforce or ZoomInfo currently think about the state of Machine Learning? What their biggest challenges are? And what they're most excited about? Well, you're about to find out!
```

<img class="grandmahugs" style="float: left;" padding="5px" width="200" src="/blog/assets/64_ml_director_insights/Omar-Rahman.jpeg"></a>  

### [Omar Rahman](https://www.linkedin.com/in/omar-rahman-4739713a/) - Director of Machine Learning at [Salesforce](https://www.salesforce.com/)

**Background:** Omar leads a team of Machine Learning and Data Engineers in leveraging ML for defensive security purposes as part of the Cybersecurity team. Previously, Omar has led data science and machine learning engineering teams at Adobe and SAP focusing on bringing intelligent capabilities to marketing cloud and procurement applications. Omar holds a Master’s degree in Electrical Engineering from Arizona State University. 

**Fun Fact:** Omar loves to play chess and volunteers his free time to guide and mentor graduate students in AI.

**Salesforce:** World's #1 customer relationship management software.
 
#### **1. How has ML made a positive impact on SaaS?**
ML has benefited SaaS offerings in many ways. Here are a few examples:

> a. Improving automation within applications: Example: A service ticket router using NLP to understand the context of the service request and routing it to the appropriate team within the organization.

Reviewer 2: 😍 love this use case! It was one of the main use cases at MonkeyLearn and it changed (for good) how customer service teams work

 
#### **1. How has ML made a positive impact on SaaS?**
ML has benefited SaaS offerings in many ways. 

> **a. Improving automation within applications:** For example, a service ticket router using NLP to understand the context of the service request and routing it to the appropriate team within the organization.

Reviewer: Given it's first mention of NLP, maybe we should put the whole name
